examples/hotrod/pkg/tracing/init.go

```diff
@@ -43,9 +44,28 @@ import (
 
 var once sync.Once
 
-// Init initializes OpenTelemetry SDK and uses OTel-OpenTracing Bridge
+// InitOTEL initializes OpenTelemetry SDK
+// to return an Otel tracer.
```

Comment:
```suggestion
// InitOTEL initializes OpenTelemetry SDK.
```
No need to explain in the comment what the function returns, it's obvious from the signature.